### PR TITLE
Refactorings; documentation

### PR DESCRIPTION
**Pull request by astaff**
In this PR:
* Cleaning up code, making it easier to read
* Adding docstrings

### CODE REVIEW

File: opentrons_sdk/containers/placeable.py [hunk 1]
```diff
@@ -57,12 +82,21 @@ def __init__(self, parent=None, properties=None):
...
+        """
+        Returns placeable by name or index
+        If slice is given, returns a list
+        """
         if isinstance(name, int) or isinstance(name, slice):
             return self.get_children_list()[name]
-        else:
+        elif isinstance(name, str):
             return self.get_child_by_name(name)
+        else:
```
Comment (levthedev) [inline opentrons_sdk/containers/placeable.py]: Slice, not sice

File: opentrons_sdk/containers/placeable.py [hunk 2]
```diff
@@ -236,23 +323,56 @@ def from_polar(self, r, theta, h):
...
+        coordinates = self.from_center(x=0, y=0, z=-1, reference=reference)
         return (self, coordinates + (0, 0, z))
 
     def top(self, z=0):
+        """
+        Returns (:Placeable, :Vector:) tuple where :Vector:
+        corresponds to the top of a :Placeable:
+
+        If :reference: :Placeable: is provided, returns
+        the :Vector: within :reference: coordiante system
```
Comment (levthedev) [inline opentrons_sdk/containers/placeable.py]: Typo in coordinate. Also, docstring mentions reference but this function doesn't take a reference

**Comment by astaff** [comment]
@levthedev fixed as per your comments. @SimplyAhmazing please give it another look when you have a moment.

**Comment by SimplyAhmazing** [comment]
@astaff this looks great! Sorry I carried away -- LGTM

**Pull request merged**